siobrultech_protocols/gem/protocol.py

```diff
@@ -193,6 +194,7 @@ def __init__(self, queue: asyncio.Queue[Packet]):
 
     def data_received(self, data: bytes) -> None:
         if self._state == ProtocolState.SENT_API_REQUEST:
+            LOG.debug(f"Received {len(data)} bytes")
```

Comment:
Dunno how useful this one is given the log in `receive_api_response`, but up to you.